archive-tar.c

```diff
@@ -463,18 +463,34 @@ static int write_tar_filter_archive(const struct archiver *ar,
 	filter.use_shell = 1;
```

Comment:
This commit message does not clarify the intention of the change.

But the other commit has the explanation for *this* commit... So let's use it here instead?